weth.md

```diff
@@ -0,0 +1,119 @@
+WETH Token
+=========
+
+The WETH token represents an ERC20 compatible version of ETH.
+
+```k
+requires "kmcd-driver.k"
+
+module WETH
+    imports KMCD-DRIVER
+
+    configuration
+      <weth>
+        <weth-stack> .List </weth-stack>
+        <weth-state>
+          <weth-totalSupply> 0    </weth-totalSupply>
+          <weth-account-id>  0    </weth-account-id>
+          <weth-balance>     .Map </weth-balance>     // mapping (address => uint)                      Address |-> Int
+          <weth-allowance>   .Map </weth-allowance>   // mapping (address => mapping (address => uint))
+        </weth-state>
+      </weth>
+
+    syntax AllowanceAddress ::= "{" Address "->" Address "}"
+ // --------------------------------------------------------
```

Comment:
Hmmm, this will probably be problematic because the same symbol is used in `dai.md`. So I think we want to declare this in `kmcd-driver.md` instead a single time, and use it in both places.